static/src/inline-svgs/icon/home.svg

```diff
@@ -0,0 +1 @@
+<svg width="15" height="18" viewBox="0 0 15 18"><path d="M7.9 0h-1L0 7v10.1l.9.9h4.2v-6.6h4.7V18h4.3l.9-1V7z"/></svg>
```

Comment:
That took the whole of digidev (and beyond) several days and resulted in at least one bug opened against Gecko: https://github.com/guardian/frontend/pull/5298 😆